src/ui/public/kibiter/menu/get_metadashboard.js

```diff
@@ -0,0 +1,55 @@
+
+export function getMetadashboard(es, kbnIndex, scope) {
```

Comment:
This function mixes functions of different levels of abstraction. I suggest to:

* Rename this function to `retrieveKibiterMenuData` or something similar.
* Move the code that searches in ElasticSearch to two different functions: `fetchMetadashboar` and `fetchProjectName`.
* Rename `styleSelected` function to `highlightMenu`.
* Call these functions in the body of this one.